src/app/__test__/Login.test.tsx

```diff
@@ -0,0 +1,8 @@
+import { expect, test } from "vitest";
```

Comment:
I'm not sure if it makes sense to have a separate directory for tests. Wdyt? IMO should be kept close to component origin.

Reply:
it is not necessary to have a separate directory for testing.so i kept it in __test__ folder but now i decided to keep it with component origin